tests/__mocks__/lodash-es/memoize.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Mock memoize by just returning the function immediately.
+ *
+ * @param {Function} callback
+ *
+ * @returns {Function}
+ */
+export default (callback) => callback;
```

Comment:
Be carful when using this mock. Memoise is a beautiful tool for making bugs and should not always be mocked away